src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { CommentType } from '../../types';
+import './Comment.scss';
+
+export function Comment({ email, name, body }) {
+  return (
+    <div className="comment">
+      <p>
+        User
+        <span className="comment__user-mail">
+          {` ${email} `}
+        </span>
+        replied:
+      </p>
+      <p className="comment__title">{name}</p>
+      <p>{body}</p>
+    </div>
+  );
+}
+
+Comment.propTypes = CommentType.isRequired;
```

Comment:
let's describe all props here

```suggestion
Comment.propTypes = {
  body: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired,
  id: PropTypes.number.isRequired,
  name: PropTypes.string.isRequired,
  postId: PropTypes.number.isRequired,
};
```

Reply:
Я не понимаю, когда мы должны выносить типы в types.js, а когда - в файл компонента